scripts/imc_preprocessing.py

```diff
@@ -13,106 +13,124 @@
 
 # %%
-# the input_data_folders_path_inputs with the ziped acquisition files for the analysis
+# the paths with the ziped acquisition files
 raw_dirs = ["../raw"]
+raw_dirs = [Path(raw_dir) for raw_dir in raw_dirs]
 
-# output for OME tiffs
-work_dir = "../analysis"
+# regular expression to select files
+file_regex = "*Patient*.zip"
 
-# panel
-panel_file = "../config/example_panel.csv"
+# panel information
+panel_file = "../raw/panel.csv"
 panel_channel_col = "Metal Tag"
 panel_keep_col = "full"
 panel_ilastik_col = "ilastik"
 
+# %% [markdown]
+# ## Specify the outputs
+#
+# You will need to specify a single folder where the output files of the pipeline are written out to (`work_dir`).
+# Within the working directory, the following sub-folder will be created:
+#
+# * `acquisitions_dir`: storing individual acquisitions as `.ome.tiff` files, panoramas as `.png` and acquisition metadata (default `analysis/ometiff`)
+# * `ilastik_dir`: storing multi-channel images in `.tiff` format for ilastik training. The channel order for each image is written out in `.csv` format (default `analysis/ilastik`). Following the CellProfiler pipelines, all files related to the ilastik segmentation approach will be stored here. 
+# * `crops_dir`: stores image crops for ilastik training after running the first CellProfiler pipeline (default `analysis/crops`)
+# * `cellprofiler_input_dir`: all files needed for CellProfiler input (default `analysis/cpinp`)
+# * `cellprofiler_output_dir`: all files written out by CellProfiler (default `analysis/cpout`)
+# * `histocat_dir`: folders containing single-channel images for histoCAT upload (default `analysis/histocat`)
+#
+# Within the `cellprofiler_output_dir` three subfolder are created storing the final images:
+#
+# * `final_images`: stores the hot pixel filtered multi-channel images containing selected channels (default `analysis/cpout/images`)
+# * `final_masks`: stores the final cell segmentation masks (default `analysis/cpout/masks`)
+# * `final_probabilities`: stores the downscaled pixel probabilities after ilastik classification (default `analysis/cpout/probabilities`)
+
 # %%
-raw_dirs = [Path(raw_dir) for raw_dir in raw_dirs]
+# working directory storing all outputs
+work_dir = "../analysis"
 work_dir = Path(work_dir)
 work_dir.mkdir(exist_ok=True)
 
-# parameters for resizing the images for ilastik
+# general output directories
 acquisitions_dir = work_dir / "ometiff"
-analysis_dir = work_dir / "tiffs"
 ilastik_dir = work_dir / "ilastik"
+crops_dir = work_dir / "crops"
 cellprofiler_input_dir = work_dir / "cpinp"
 cellprofiler_output_dir = work_dir / "cpout"
 histocat_dir = work_dir / "histocat"
 
+# Final output directories
```

Comment:
Following naming conventions in this notebook, these should have a `_dir` suffix